lib/netpacket/netpacket_write.c

```diff
@@ -80,7 +80,7 @@ netpacket_write_file(NETPACKET_CONNECTION * NPC,
 	packetbuf[8] = class;
 	memcpy(&packetbuf[9], name, 32);
 	memcpy(&packetbuf[41], nonce, 32);
-	be32enc(&packetbuf[73], buflen);
+	be32enc(&packetbuf[73], (uint32_t)buflen);
```

Comment:
Given that we've validated `buflen` a few lines earlier, I'd go with the cast to `uint32_t`.